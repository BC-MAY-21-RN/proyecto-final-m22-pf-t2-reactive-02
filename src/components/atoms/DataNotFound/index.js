import React from 'react';
import {Text, View, Image} from 'react-native';
import styles from './styles';

export default function DataNotFound({title, text}) {
  return (
    <View style={styles.container}>
      <Text style={styles.text}>{title}</Text>
      <Image
        style={styles.image}
        source={{
          uri: 'https://res.cloudinary.com/dzdgpwtox/image/upload/w_600,c_scale/v1622293248/designer-tool-uploads/bucket_4052/1622293241335.png',
        }}
      />
      <Text style={styles.info}>{text}</Text>
    </View>
  );
}
