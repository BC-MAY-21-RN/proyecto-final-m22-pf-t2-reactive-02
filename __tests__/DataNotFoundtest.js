import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import DataNotFound from '../src/components/atoms/DataNotFound';

describe('Test from DataNotFound component', () => {
  it('DataNotFound snapshot test', () => {
    const renderDataNotFound = renderer
      .create(<DataNotFound text={''} title={''} />)
      .toJSON();
    expect(renderDataNotFound).toMatchSnapshot();
  });
});
