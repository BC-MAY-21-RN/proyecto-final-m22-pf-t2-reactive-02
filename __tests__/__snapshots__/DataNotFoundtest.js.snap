// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test from DataNotFound component DataNotFound snapshot test 1`] = `
<View
  style={
    Object {
      "justifyContent": "center",
      "marginTop": 100,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "center",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginTop": 20,
      }
    }
  />
  <Image
    source={
      Object {
        "uri": "https://res.cloudinary.com/dzdgpwtox/image/upload/w_600,c_scale/v1622293248/designer-tool-uploads/bucket_4052/1622293241335.png",
      }
    }
    style={
      Object {
        "alignSelf": "center",
        "height": 180,
        "marginTop": 20,
        "width": 180,
      }
    }
  />
  <Text
    style={
      Object {
        "fontSize": 18,
        "marginTop": 20,
        "textAlign": "center",
      }
    }
  />
</View>
`;
